src/main/java/ama/post/PostController.java

```diff
@@ -48,6 +46,7 @@ public String postAMASubmission(@ModelAttribute("submissionPost") SubmissionPost
         } else {
             post.setOp(user);
             submissionPostRepository.save(post);
+            System.out.println(post.getVotingCloses());
```

Comment:
Did you mean to keep this? Seems like you only wanted to check something. Might want to remove if it's not necessary.